python/grass/pygrass/tests/benchmark.py

```diff
@@ -343,7 +345,7 @@ def print_test(testdict):
 TXT = """
 {% for region in regions %}
 {{ '#'*60 }}
-### Benchmark cols = {{ region.cols }} rows = {{ region.rows}} cells = {{ region.cells }}
+### Benchmark cols = {{region.cols}} rows = {{region.rows}} cells = {{region.cells}}
```

Comment:
The spaces around the variables are part of jinja template syntax, so we shouldn't remove them.
```suggestion
# Benchmark cols = {{ region.cols }} rows = {{ region.rows }} cells = {{ region.cells }}
```